src/control/server/README.md

```diff
@@ -163,205 +163,65 @@ In order to run the tool to perform administrative tasks, build and run the `dao
 
-Help Options:
-  -h, --help         Show this help message
+#### SCM Firmware Update
 
-Available commands:
-  kill-rank  Terminate server running as specific rank on a DAOS pool (aliases: kr)
+#### SCM Burn-in Validation
 
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/daos_shell service kill-rank --help
-Usage:
-  daos_shell [OPTIONS] service kill-rank [kill-rank-OPTIONS]
+### NVMe management capabilities
 
-Application Options:
-  -l, --hostlist=      comma separated list of addresses <ipv4addr/hostname:port> (default: localhost:10001)
-  -f, --hostfile=      path of hostfile specifying list of addresses <ipv4addr/hostname:port>, if specified takes preference over HostList
-  -o, --config-path=   Client config file path
+Operations on NVMe SSD devices are performed using [go-spdk bindings](https://github.com/daos-stack/go-spdk) to issue commands over the SPDK framework.
 
-Help Options:
-  -h, --help           Show this help message
-
-[kill-rank command options]
-      -r, --rank=      Rank identifying DAOS server
-      -p, --pool-uuid= Pool uuid that rank relates to
-```
+The DAOS data plane utilises NVMe devices as block storage using the SPDK "[blobstore](https://spdk.io/doc/blob.html)" abstraction, creation and management of the blobstore and blobs is performed by the data plane.
 
-</p>
-</details>
-
-<details>
-<summary>Example output from invoking "storage list" subcommand</summary>
-<p>
-
-```
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/daos_shell -l boro-44:10001,boro-45:10001 storage list
-Active connections: [boro-45:10001 boro-44:10001]
-
-Listing NVMe SSD controller and constituent namespaces on connected storage servers:
-boro-44:10001:
-- id: 0
-  model: 'INTEL SSDPED1K375GA '
-  serial: 'PHKS73350016375AGN  '
-  pciaddr: 0000:81:00.0
-  fwrev: E2010324
-  namespace:
-  - id: 1
-    capacity: 375
-boro-45:10001:
-- id: 0
-  model: 'INTEL SSDPED1K375GA '
-  serial: 'PHKS7335006W375AGN  '
-  pciaddr: 0000:81:00.0
-  fwrev: E2010420
-  namespace:
-  - id: 1
-    capacity: 375
+#### NVMe Controller and Namespace Discovery
 
+Device details for any discovered NVMe SSDs accessible through SPDK on the storage server will be returned.
 
-Listing SCM modules on connected storage servers:
-boro-44:10001: []
-boro-45:10001: []
-```
+The following animation illustrates starting the control server and using the management shell to view the NVMe Namespaces discovered on a locally available NVMe Controller (assuming the quickstart guide instructions have already been performed):
 
-</p>
-</details>
+![Demo: List NVMe Controllers and Namespaces](/doc/graph/daosshellnamespaces.svg)
 
-<details>
-<summary>Example output when listing storage in interactive shell mode</summary>
-<p>
+#### NVMe Format
 
-```bash
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/daos_shell
-Active connections: [localhost:10001]
+Format can be triggered through the management tool at DAOS system installation time.
```

Comment:
This is a bit unclear on how the management tool gets triggered to format NVMe. Also, is the only time NVMe can be formatted strictly at system installation time?